### PR TITLE
9.16 RN

### PR DESCRIPTION
**Pull request by Adam-Dupaski**

### CODE REVIEW

File: content/en/docs/releasenotes/studio-pro/9/9.16.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+* We fixed an issue where in restricted markets, Studio Pro still tried to send analytics.
+* We fixed an issue where changing the `selectableObjects` attribute of an association property in the XML definition of a pluggable widget did not require the widget to be updated, causing the widget to misbehave in the client.
+* We fixed an issue where an error during file retrieval caused Mendix Runtime to close the connection before sending an error response.
+* We fixed an issue where MxAssist Performance Bot displayed an incorrect total count in a panel icon when some recommendations had been suppressed.
+* We fixed an issue where read-only text controls received focus on tab navigation.
+* We fixed an issue where changes to the page layout of a workflow admin page and a user task page were not applied after a hot-redeploy of the page changes. [**TW: "hot-redeploy" in two instances?**]
+* We fixed an issue with the MxAssist Performance Bot **MXP013** best practice where some access rules caused the inspection to end without outputting results.
+* We fixed the tab navigation behavior in tree controls.
+
```
Comment (ghost) [inline content/en/docs/releasenotes/studio-pro/9/9.16.md:72]: Hi @Adam-Dupaski I would like to add the following (forgotten) line to the release notes:

`* We fixed an issue with Team Server: Git deployments. (Tickets 151107, 152169, 152222, 152672)`

I'll let my SDM know so he could approve.

Comment (MxMarcel) [reply content/en/docs/releasenotes/studio-pro/9/9.16.md:72]: SDM says ok ;-)

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/studio-pro/9/9.16.md:72]: Added, but do we need any more detail? There's not a lot of info in the note @A-Ovchinnikov-mx @MxMarcel

Comment (ghost) [reply content/en/docs/releasenotes/studio-pro/9/9.16.md:72]: There's not a lot of additional context either, deployments were just failing 🤷 The other stuff is mostly technical details I presume.

File: content/en/docs/releasenotes/studio-pro/9/9.16.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+title: "9.16"
+url: /releasenotes/studio-pro/9.16/
+parent: "9"
```
Comment (NatasaKralj) [inline content/en/docs/releasenotes/studio-pro/9/9.16.md]:
```suggestion
```

Comment (NatasaKralj) [reply content/en/docs/releasenotes/studio-pro/9/9.16.md]: `parent` parameters are no longer used in front matter

**Pull request merged**